### ISSUE TITLE
Create a daemon for coordinating with Spotify API and updating the InkyWHAT display

### ISSUE THREAD

**Issue by rclim95** [rclim95/PlayingWHAT#2]
This daemon should be responsible for the following:

* [x] Read from a configuration file that specifies how to read from the Spotify API
* [x] Communicating with the Spotify API to get what's playing
* [x] Update the InkyWHAT display with the information about the track that's currently being played
* [x] Handle local messages that get sent to the daemon via IPC (i.e., Unix sockets), including:
  * [x] Stopping when a `stop` message is sent
  * [x] Reload the configuration saved in memory when a `reload` message is sent

In other words, the daemon will be the powerhouse of the PlayingWHAT project. 😅
